softlayer/vm/softlayer_creator.go

```diff
@@ -56,11 +76,20 @@ func (c SoftLayerCreator) Create(agentID string, stemcell bslcstem.Stemcell, clo
 			GlobalIdentifier: stemcell.Uuid(),
 		},
 
-		SshKeys:           cloudProps.SshKeys,
-		HourlyBillingFlag: true,
-
-		// Needed for ephemeral disk
-		LocalDiskFlag: true,
+		SshKeys: cloudProps.SshKeys,
+
+		//HourlyBillingFlag: true,
```

Comment:
When you fix code, you should just remove, don't comment old code and leave it since it confuses things in future.